syntax = "proto2";
package DB.Protos;

message ProtoHiveFile
{
    optional uint32 format = 1;
    optional string file_path = 2;
    optional uint64 file_size = 3;
    optional string storage = 4;
    optional string partition_id = 5;
}

message ProtoHiveFiles
{
    repeated ProtoHiveFile files = 1;
    optional string sd_url = 2;
}

